bindings/gumjs/gumquickmemory.c

```diff
@@ -834,6 +837,28 @@ gum_quick_memory_write (JSContext * ctx,
   return result;
 }
 
+GUMJS_DEFINE_FUNCTION (gum_quick_memory_read_volatile_array)
+{
+  gpointer address;
+  gssize length = -1;
```

Comment:
Variable should not be initialized, as it's guaranteed to be initialized by `_gum_quick_args_parse()` since the argument is required.